plugins/locale-observer.js

```diff
@@ -8,7 +8,11 @@ export default function (context) {
 }
 
 function _setLocaleCookie(locale, isDev) {
-  const canonicalName = Intl.getCanonicalLocales(locale)?.[0]
+  let canonicalName = Intl.getCanonicalLocales(locale)?.[0]
+
+  if (canonicalName === 'en-GB') {
```

Comment:
Question : est-ce qu'on peut faire cette modif de suite ? Quand est-ce qu'est appelé ce plugin ?

Reply:
Quand j'ai fais le test fonctionnel sur pix site et que j'ai changé la locale en en-GB, en allant sur pix pro, la valeur est de suite changé à 'en', c'est lié au locale-observer ?

Reply:
Le plugin est appelé :
- Lorsque tu vas sur un path qui ne contient pas de locale et qui fonctionne (https://pix.org/ du coup 😄 ). Le module i18n va déterminer la locale de l'utilisateu puis  appeler le `onBeforeLanguageSwitch`  dans le locale-observer ce qui va setter la valeur. 
- Si tu changes de locale manuellement avec le composant LocaleSwitcher.

Dans ces 2 cas, si on avait comme locale `en-GB`, il aura comme nouvelle valeur `en`.

Par contre, si tu vas sur une url qui contient déjà une locale en path (ex: https://pix.org/en-gb/the-tests), alors le switch n'est pas trigger et l'utilisateur gardera `en-GB` comme valeur !

Reply:
Cool, merci :)